### PR TITLE
Setup CircleCI

### PR DESCRIPTION
**Pull request by ehrmann**
Addresses #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Requirements
 
-  * Ruby 1.9.3 or above.
+  * Ruby 2.0.0 or above.
```
Comment (ehrmann) [inline README.md]: Looks like 1.9.3 support was dropped 4 years ago: https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+version: 2
+jobs:
+  build:
+    parallelism: 3 # run three instances of this job in parallel
+    docker:
+      - image: circleci/ruby:2.4.2-jessie-node
```
Comment (ehrmann) [inline .circleci/config.yml]: 2.4 is the [oldest supported version](https://en.wikipedia.org/wiki/Ruby_(programming_language)#Table_of_versions)

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        environment:
+          BUNDLE_JOBS: 3
+          BUNDLE_RETRY: 3
+          BUNDLE_PATH: vendor/bundle
+    steps:
+      - checkout
+
+      - run:
+          name: Which bundler?
+          command: bundle -v
```
Comment (garylee1) [inline .circleci/config.yml]: is this just debug output?

Comment (ehrmann) [reply .circleci/config.yml]: (For the task) I think. Surprisingly, `-v` is version, not verbose for `bundler`. I based this on a simple config circle hosted.

**Pull request merged**